packages/config/src/layer2s/index.ts

```diff
@@ -1,18 +1,19 @@
+import { HOMEPAGE_MILESTONES } from './common'
+import { Layer2 } from './types'
 import { apex } from './apex'
 import { arbitrum } from './arbitrum'
 import { aztec } from './aztec'
 import { aztecconnect } from './aztecconnect'
 import { base } from './base'
 import { bobanetwork } from './bobanetwork'
-import { HOMEPAGE_MILESTONES } from './common'
-import { linea } from './consensys'
 import { dydx } from './dydx'
 import { fuelv1 } from './fuelv1'
 import { gluon } from './gluon'
 import { hermez } from './hermez'
 import { immutablex } from './immutablex'
 import { layer2finance } from './layer2finance'
 import { layer2financezk } from './layer2financezk'
+import { linea } from './consensys'
```

Comment:
We have an eslint rule for imports. Changing them manually will cause the CI to fail